examples/remix-with-typescript/package.json

```diff
@@ -9,22 +9,22 @@
     "start": "remix-serve build"
   },
   "dependencies": {
-    "@emotion/cache": "latest",
-    "@emotion/react": "latest",
-    "@emotion/server": "latest",
-    "@emotion/styled": "latest",
-    "@mui/material": "latest",
-    "@remix-run/react": "latest",
-    "@remix-run/serve": "latest",
-    "react": "latest",
-    "react-dom": "latest",
-    "remix": "latest"
+    "@emotion/cache": "^11.10.3",
```

Comment:
Please unpin all the dependencies, it's important for us to keep the example up to date. (meaning we prefer the demo to break when there is a BC release so we can then quickly know about it to fix it).